ktor-core-tests/test/org/jetbrains/ktor/tests/http/StatusPageTest.kt

```diff
@@ -23,7 +23,9 @@ class StatusPageTest {
                 call.respond(HttpStatusCode.NotFound)
             }
             handleRequest(HttpMethod.Get, "/foo").let { call ->
+                assertEquals(HttpStatusCode.NotFound, call.response.status(), "Actual status must be kept")
```

Comment:
This test failed (handler of `statusFile` responded with `200 Ok`) and caused the changes in `StatusPages.Configuration.statusFile`